README.md

```diff
@@ -28,8 +28,11 @@ After completing these steps, you can find the compiled VM in `bin/cairo-vm`.
 
 ### Run The VM
 
-To run the VM you need first to have a compiled Cairo file using the Cairo compiler at [cairo-lang](https://github.com/starkware-libs/cairo-lang).
-Install it with the following command:
+To run the VM you need to have a compiled Cairo file using the Cairo compiler at [cairo-lang](https://github.com/starkware-libs/cairo-lang).
```

Comment:
I know this is old, but I would say: _"...file using the Cairo **Zero** compiler..."_

Reply:
I like it it's more precise
Thanks !